package.json

```diff
@@ -56,6 +56,7 @@
     "@nomiclabs/hardhat-truffle5": "^2.0.1",
     "@nomiclabs/hardhat-web3": "^2.0.0",
     "@openzeppelin/contracts": "^4.3.3",
```

Comment:
question: Now the Governance is upgradable, is the `openzeppelin/contracts` dependency still needed?